src/lib/components/documents/ResultsList.svelte

```diff
@@ -87,6 +87,7 @@
 </script>
 
 <div class="container">
+  <slot />
```

Comment:
While we're making this component more configurable, how about a `<slot name="start" />` at the top and a `<slot name="end" />` at the bottom of the list?